### PR TITLE
api: split graphql schemas

### PR DESCRIPTION
**Pull request by bauer01**
## Description

< Some description of what changes are included in this PR >

### Major Changes
 - :rocket: < Name of the task > [WT-XXX](link_to_the_task)

### Bug Fixes
 - :ambulance: < Name of the task > [WT-XXX](link_to_the_task)

### Minor Tweaks
 - :lipstick: < Name of the task > [WT-XXX](link_to_the_task)

### For feature development:
- [x] Have you tested your code?
- [ ] Did you commit only code related to your feature?
- [x] Are you sure you did not break any other feature?
- [x] Have you checked your PR before you assigned it to reviewer(s)?
- [x] Are you sure your feature does not contain any hacks?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
 	},
 	"scripts": {
 		"env": "env-cmd -f ./.env",
+		"deploy": "lerna --stream --scope='@be-prioritized/*' run deploy -- -- -p .serverless && lerna --stream --scope='@be/*' run deploy -- -- -p .serverless && lerna --stream --scope='api' run deploy -- -- -p .serverless && lerna --stream --scope='frontend' run deploy",
```
Comment (FilipPyrek) [inline package.json:10]: Why did you add this? I don't think we need `deploy` in the root folder.

File: api/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
 		"ci:test": "NODE_ENV=test jest --coverage --detectOpenHandles --forceExit --passWithNoTests"
 	},
 	"devDependencies": {
+		"@graphql-tools/merge": "^6.2.14",
```
Comment (FilipPyrek) [inline api/package.json:26]: Where is this being used?

**Comment by FilipPyrek** [comment]
So the issue with appsync simulator is resolved @bauer01 ?

**Comment by bauer01** [comment]
> So the issue with appsync simulator is resolved @bauer01 ?



> So the issue with appsync simulator is resolved @bauer01 ?

@FilipPyrek yes, watchman reinstall and then simulator upgrade helped 😉

**Review by FilipPyrek — APPROVED** [review]
